src/main/java/seedu/address/model/person/Address.java

```diff
@@ -12,10 +12,10 @@ public class Address {
     public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
-     * otherwise " " (a blank string) becomes a valid input.
+     * As the address is now optional, the validation regex accepts a blank address
```

Comment:
Maybe the comment shouldn't include "now"? 

Maybe "Validation regex accepts blank address string because address is optional" would be easier to understand for future devs who have no background knowledge that this was from AB3?